optuna/samplers/_cmaes.py

```diff
@@ -16,13 +16,15 @@
 from optuna.distributions import BaseDistribution
 from optuna.exceptions import ExperimentalWarning
 from optuna.samplers import BaseSampler
+from optuna.storages import BaseStorage
 from optuna.trial import FrozenTrial
 from optuna.trial import TrialState
 
 
 _logger = logging.get_logger(__name__)
 
 _EPS = 1e-10
+_SYSTEM_ATTR_MAX_LENGTH = 2048
```

Comment:
This should be less than 2046 characters because optimizer_str is stored after `json.dumps()`.

```
sqlalchemy.exc.DataError: (MySQLdb._exceptions.DataError) (1406, "Data too long for column 'value_json' at row 1")
[SQL: INSERT INTO trial_system_attributes (trial_id, `key`, value_json) VALUES (%s, %s, %s)]
[parameters: (46, 'cma:optimizer:0', '"80049528050000000000008c0a636d6165732e5f636d61948c03434d419493942981947d94288c065f6e5f64696d944b028c085f706f7073697a65944b068c035f6d75944b038c075f6d ... (1751 characters truncated) ... 16c75657394682f68314b008594683387945294284b014b288594681189424001000000000000000000500000000000000050997218541cbe26c051d339ed0fa99440010000000000000"')]
(Background on this error at: http://sqlalche.me/e/9h9h)
```